gene_finder.py

```diff
@@ -45,9 +52,12 @@ def get_reverse_complement(dna):
     >>> get_reverse_complement("CCGCGTTCA")
     'TGAACGCGG'
     """
-    # TODO: implement this
-    pass
+    compliment = ""
+    for f in dna:
```

Comment:
Try using variable names that are more descriptive than a random letter - it helps with code readability.